src/conf/startup.properties

```diff
@@ -331,7 +331,8 @@ prism.configstore.listeners=org.apache.falcon.entity.v0.EntityGraph,\
 
 # Setting monitoring plugin, if SMTP parameters is defined
 #*.monitoring.plugins=org.apache.falcon.plugin.DefaultMonitoringPlugin,\
-#                     org.apache.falcon.plugin.EmailNotificationPlugin
+#                     org.apache.falcon.plugin.EmailNotificationPlugin,\
+#                     org.apache.falcon.plugin.FalconDBNotificationPlugin
```

Comment:
We can add one line comments in the properties file but the functionality and scenario should go the twiki.Let me know your thoughts will work accordingly.